### PR TITLE
Include access object for public elements

### PR DESCRIPTION
**Pull request by pzuraq**
Per #450, updates context to include accessor methods for public elements.

### CODE REVIEW

**Review by Jack-Works — APPROVED** [review]
Support this change

File: README.md [hunk 1]
```diff
@@ -965,8 +912,8 @@ class Container {
 
     const { public, private } = instance[Symbol.metadata][INJECTIONS];
 
-    for (let key in public) {
-      instance[key] = this.lookup(public[key].injectionKey);
+    for (let { injectionKey, set } in public) {
+      set.call(instance, this.lookup(injectionKey))
```
Comment (ljharb) [inline README.md:916]: for data properties, presumably `set` would do a [[Set]], iow, be `function (v) { this[prop] = v; }`?

Comment (pzuraq) [reply README.md:916]: Yes, correct

**Review by ljharb — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (ljharb) [reply README.md:916]: built-in lenses, i like it, picasso

**Pull request merged**